src/cppcache/src/statistics/HostStatHelperWin.cpp

```diff
@@ -535,21 +535,21 @@ uint32 HostStatHelperWin::getInt32Value(PPERF_COUNTER_DEFINITION PerfCntr,
   }
 
   if (PerfCntr->CounterSize == 4) {
-    uint32* lptr;
-    lptr = (uint32*)((char*)PerfCntrBlk + PerfCntr->CounterOffset);
+    uint32_t* lptr;
+    lptr = (uint32_t*)((char*)PerfCntrBlk + PerfCntr->CounterOffset);
```

Comment:
What is this really trying to do? Maybe fix it?

C++11 style casting changes at least.